src/WireMock/Client/HttpWait.php

```diff
@@ -4,7 +4,7 @@
 
 class HttpWait
 {
-    public function waitForServerToGive200($url, $timeoutSecs = 10)
+    public function waitForServerToGive200($url, $timeoutSecs = 10, $debug = false)
```

Comment:
Would it make more sense to default this to true? It means there's no change in the current behaviour, which is nice, but I also think it makes sense: we expect WireMock to come up in the vast majority of cases, so I think it's more useful to be verbose here by default (and allow the option to suppress that debug trace if someone finds it annoying in their particular case).